### PR TITLE
Bump versions for 2020-01-11 release

### PR DESCRIPTION
**Pull request by azabbasi**
Release notes:

# Microsoft.Azure.Devices.Client.1.30.0
- Remove unreferenced exceptions and their mappings
     * DeviceAlreadyExistsException
     * DeviceDisabledException
     * IotHubNotFoundException

# Microsoft.Azure.Devices.Provisioning.Service 1.7.0
- Add APIs to get device attestation mechanism

## Bug fixes:
- Fix GetFileUploadSasUriAsync API when using certificate authentication methods (https://github.com/Azure/azure-iot-sdk-csharp/issues/1527)

### CODE REVIEW

File: iothub/device/src/Microsoft.Azure.Devices.Client.csproj [hunk 1]
```diff
@@ -36,7 +36,7 @@
     <Description>Device SDK for Azure IoT Hub</Description>
     <PackageIcon>nugetIcon.png</PackageIcon>
     <PackageProjectUrl>https://github.com/Azure/azure-iot-sdk-csharp</PackageProjectUrl>
-    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <Copyright>Â© Microsoft Corporation. All rights reserved.</Copyright>
```
Comment (drwill-ms) [inline iothub/device/src/Microsoft.Azure.Devices.Client.csproj:39]: ?

**Comment by drwill-ms** [comment]
> Release notes:
> 
> # Microsoft.Azure.Devices.Client.1.30.0
> * Remove unreferenced Exceptions and their mappings
>   
>   * DeviceAlreadyExistsException
>   * DeviceDisabledException
>   * IotHubNotFoundException
> 
> # Microsoft.Azure.Devices.Provisioning.Service 1.7.0
> * Add APIs to get attestation mechanism
> 
> ## Bug fixes:
> * Fix GetFileUploadSasUriAsync API when using certificate authentication methods (#1527)

Can you point me to the DPS "attestation mechanism" fix?

File: iothub/device/src/Microsoft.Azure.Devices.Client.csproj (hunk 1 above)
Comment (azabbasi) [reply iothub/device/src/Microsoft.Azure.Devices.Client.csproj:39]: No idea how that got in there ! it's generated by the script ... I will investigate

**Comment by azabbasi** [comment]
> > Release notes:
> > # Microsoft.Azure.Devices.Client.1.30.0
> > 
> > * Remove unreferenced Exceptions and their mappings
> >   
> >   * DeviceAlreadyExistsException
> >   * DeviceDisabledException
> >   * IotHubNotFoundException
> > 
> > # Microsoft.Azure.Devices.Provisioning.Service 1.7.0
> > 
> > * Add APIs to get attestation mechanism
> > 
> > ## Bug fixes:
> > 
> > * Fix GetFileUploadSasUriAsync API when using certificate authentication methods (#1527)
> 
> Can you point me to the DPS "attestation mechanism" fix?

https://github.com/Azure/azure-iot-sdk-csharp/pull/1526

**Review by drwill-ms — APPROVED** [review]

**Review by vinagesh — APPROVED** [review]

**Review by abhipsaMisra — APPROVED** [review]

**Pull request merged**